G/gecko/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "gecko"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/LLNL/gecko.git", "490ab7d9b7b4e0f007e10d2af2b022b96d427fee")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/gecko/
+mkdir build
+cd build
+cmake .. -DCMAKE_INSTALL_PREFIX=$WORKSPACE/destdir/
+cmake --build . --config Releaese
+make install
```

Comment:
```suggestion
cmake -S ./ -B build .. \
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
    -DCMAKE_INSTALL_PREFIX=${prefix} \
    -DCMAKE_BUILD_TYPE=Release

cmake --build build --parallel ${nproc}
cmake --install build
```
You need to follow the guidelines here for the CMake builds to make sure the compilers are correct: https://docs.binarybuilder.org/stable/build_tips/#CMake-builds.

This is also slightly cleaner when done this way.

Reply:
Good point. Included now.